test/types/index.ts

```diff
@@ -15,6 +15,10 @@ const span = agent.currentSpan
 if (span) span.end()
 const traceparent = agent.currentTraceparent
 if (traceparent) traceparent.split('-')
+const currentTraceIds = agent.currentTraceIds
+let traceId = currentTraceIds['trace.id'] || ''
+traceId += '-' + typeof currentTraceIds['transaction.id'] !== 'undefined' ? 
+  currentTraceIds['transaction.id'] : currentTraceIds['span.id']
```

Comment:
The tiniest of nits - just aligning with how things have previously been structured elsewhere in the repo.

```suggestion
traceId += '-' + currentTraceIds['transaction.id'] === undefined
  ? currentTraceIds['transaction.id']
  : currentTraceIds['span.id']
```